Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -60,6 +60,12 @@ struct GenerateCommand: AsyncParsableCommand, HasTrackableParameters {
     )
     var ignoreCache: Bool = false
 
+    @Option(
+        name: [.customLong("skip-cache")],
+        help: "A list of targets which will not use cached binaries."
```

Comment:
This only makes sense if sources is empty, right? Otherwise you would just pass these there.
If yes, I would add a validation to check they are not both set
```suggestion
        help: "A list of targets which will not use cached binaries when using default `sources` list."
```

Reply:
Thanks for your feedback.
That's right. I added a validation to check that both 'sources' and '--skip-cache' are not both set.